sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/src/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents.csproj

```diff
@@ -3,6 +3,7 @@
     <AssemblyTitle>CloudNative CloudEvents support for Azure.Messaging.EventGrid library</AssemblyTitle>
     <Description>This library allows the CloudEvent model from CloudNative.CloudEvents to be published using the Azure Event Grid client library.</Description>
     <Version>1.0.0</Version>
+    <IsClientLibrary>true</IsClientLibrary>
```

Comment:
It shouldn't really matter;  that condition should only set the attribute to `true`,  not override the project if it is also explicitly setting it.   Since the project value is the last that gets evaluated, that should take precedence, no?

Reply:
The csproj and the build.props are setting it to the same thing with this change, so I don't think this change will have an effect one way or another. I will remove it.